sorc/global_cycle.fd/land_increments.f90

```diff
@@ -18,7 +19,19 @@ module land_increments
                                                      !! copied from GFS_typedefs.F90
 
     ! control state for soil analysis:
-    integer, parameter       :: lsoil_incr=3 !< number of layers to add incrments to
+
+    ! Originally lsoil_incr=3.
```

Comment:
Replace this comment with something like: 

"added check preventing soil (temp? moisture?) updates to layers 3 and 4, due to over sensitivity of permafrost in deeper layers" (or something like that - it just needs 1 or 2 lines. You could also put this comment where the check is made.